### ISSUE TITLE
[问题]我用Qwen-14B    4bit量化微调，官方脚本。 loss 下降到多少合适？ 我降到0.3 了，感觉效果也一般； 另外对数据集的大小建议是多少，我现在是1W 条数据。

### ISSUE THREAD

**Issue by skyantao** [QwenLM/Qwen#578]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by liudayiheng** [comment]
loss 0.3已经很低了，可以尝试模型是否能输出或补全train set的样本。另外下游任务是否和sft数据有gap？

数据量也是根据下游场景来看的，一般1w-10w级别比较常见

**Comment by skyantao** [comment]
谢谢

**Comment by EfoodCN** [comment]
能输出或补全train set的样本是代表差不多过拟合了吗，还是说拟合的刚好啊
